example/demo.py

```
@@ -20,19 +22,22 @@ async def on_download_error(trigger: aioaria2.Aria2WebsocketClient, data: dict):
 async def on_download_complete(trigger: aioaria2.Aria2WebsocketClient, data: dict):
     print("Congratulations! You have successfully across the great wall")
 
+async def on_download_pause(trigger: aioaria2.Aria2WebsocketClient, data: dict):
+    print(f"paused: {data["params"][0]["gid"]}")
+
 
 async def main():
     client: aioaria2.Aria2WebsocketClient = await aioaria2.Aria2WebsocketClient.new(
-        "http://127.0.0.1:6800/jsonrpc",
+        HOST,
         token="token",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token"

(S106)